tables/TaQL/ExprAggrNodeArray.cc

```diff
@@ -68,25 +68,25 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
     return itsFunc->isLazy();
   }
 
-  CountedPtr<TableExprGroupFuncBase> TableExprAggrNodeArray::doMakeGroupAggrFunc()
+  std::shared_ptr<TableExprGroupFuncBase> TableExprAggrNodeArray::doMakeGroupAggrFunc()
   {
     if (funcType() == TableExprFuncNode::gexpridFUNC) {
-      return new TableExprGroupExprId(this);
+      return std::shared_ptr<TableExprGroupFuncBase> (new TableExprGroupExprId(this));
```

Comment:
Optional: Use `return std::make_shared<...>(...);` in this function.

Reply:
Done